Automattic/wp-calypso
# Signup: Add an abtest for post signup landing page.

This PR will add an abtest to encourage free plan users to upgrade their plan for better SEO.

See p5uIfZ-6N7-p2 for more detail.

## How to test

1. Set abtest _postSignupUpgradeScreen_ to _modified_.
2. Create a new website and follow the steps with free plan.
3. You should be able to see the new post signup screen like the following.

![screen_shot_2017-08-10_at_23_05_45](https://user-images.githubusercontent.com/212034/29174295-d78e955e-7e20-11e7-9c7c-0c4c903ba3d0.png)
